cypress/integration/unauthenticated_spec.js

```diff
@@ -19,4 +19,67 @@ describe('The Login Page', () => {
   it('should match image snapshot', () => {
     cy.visit('/login').then(() => cy.get('div.login').toMatchImageSnapshot());
   });
+
+  it('should authenticate and redirect', () => {
```

Comment:
Does async/await work with these tests? Because that would be pretty cool and have better readability